benches/bin/mvcctxn.rs

```diff
@@ -0,0 +1,188 @@
+extern crate rand;
+
+use std::sync::atomic::{ATOMIC_U64_INIT, AtomicU64, Ordering};
+
+use tikv::storage::{new_local_engine, Engine, Key, Modify, Mutation, Options, SnapshotStore,
+                    Statistics, ALL_CFS, TEMP_DIR};
+use tikv::storage::mvcc::MvccTxn;
+use kvproto::kvrpcpb::{Context, IsolationLevel};
+
+use super::print_result;
+use test::BenchSamples;
+
+use rand::Rng;
+
+
+#[inline]
+fn next_ts() -> u64 {
+    static CURRENT: AtomicU64 = ATOMIC_U64_INIT;
+    CURRENT.fetch_add(1, Ordering::SeqCst)
+}
+
+#[inline]
+fn to_bytes(value: u64) -> Vec<u8> {
+    let mut result = Vec::new();
+    // big endian
+    for shift in (0..8).rev() {
+        result.push(((value >> shift) & 0xff) as u8);
+    }
+    result
+}
+
+fn create_row_key(tableid: u64, rowid: u64) -> Vec<u8> {
```

Comment:
please refer https://github.com/pingcap/tikv/blob/master/src/raftstore/store/keys.rs#L68 to see how we optimize the key creation.